SonarQube.Common/ProcessRunnerArguments.cs

```diff
@@ -36,7 +36,7 @@ public class ProcessRunnerArguments
         /// Strings that are used to indicate arguments that contain
         /// sensitive data that should not be logged
         /// </summary>
-        public static readonly ISet<string> SensitivePropertyKeys = new HashSet<string>
+        public static readonly IEnumerable<string> SensitivePropertyKeys = new HashSet<string>
```

Comment:
@valhristov has refactored this code in his pull request (moved the field to another class and hidden it).
Here, I've just made the minimum change required to stop the issue from reporting.